core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -1125,3 +1128,48 @@ class DynamicProducerStateManagerConfig(val producerStateManagerConfig: Producer
   override def reconfigurableConfigs: Set[String] = ProducerStateManagerConfig.RECONFIGURABLE_CONFIGS.asScala
 
 }
+
+class DynamicRemoteLogManagerConfig(server: KafkaBroker) extends BrokerReconfigurable with Logging {
```

Comment:
Perhaps keep it is `DynamicRemoteLogConfig` so that we can add other remote storage related configs here. This will also align with RemoteLogConfig sub-class in LogConfig.java